src/dstack/_internal/core/backends/aws/compute.py

```diff
@@ -133,9 +133,14 @@ def run_job(
             {"Key": "dstack_user", "Value": run.user},
         ]
         try:
+            disk_size_mib = (
+                instance_offer.instance.resources.disk.size_mib
+                or job.job_spec.requirements.disk.size_mib
+            )
+            disk_size = round(disk_size_mib / 1024) if disk_size_mib else 100
```

Comment:
AWS will fail if disk size is less than AMI size (100GB for now)

Why do we add a default value here and not in Profile, similar to CPU and RAM?